Package.swift

```diff
@@ -42,5 +51,8 @@ let package = Package(
         .testTarget(
                 name: "RouterSpec",
                 dependencies: ["Router", "Quick", "Nimble"]),
+        .testTarget(
```

Comment:
is this RouteSpec testTarget a duplicate of the one above?

Reply:
I have Route and Router in 2 separate folders. They used to be all together but I ran into circular dependency issues because in Swift we import the whole folder, not a particular file.

Reply:
Oh gotcha, `Route` and `Router` look really similar 😊